src/core/doe_commercial_reference_building_loads.jl

```diff
@@ -188,11 +189,11 @@ function built_in_load(
     else
         boiler_efficiency = 1.0
     end
-    datetime = DateTime(year, 1, 1, 1)
-    for ld in normalized_profile
-        month = Month(datetime).value
```

Comment:
This was previously starting at 1am, and was thus resulting in the last timestep of each month being incorrectly assigned to the next month (e.g., ts 744 was being assessed as month 2 and multiplied by the month 2 scalar)